### PR TITLE
Fixes #15547: Empty keyhashes in nodeslist.json on Linux nodes

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/15547

### CODE REVIEW

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala [hunk 1]
```diff
@@ -323,15 +323,18 @@ final class AsyncDeploymentActor(
 
         case Full(nodeIds) =>
           logger.info(s"Successful policy update '${id}' [started ${startTime.toString(timeFormat)} - ended ${endTime.toString(timeFormat)}]")
+          println("prout")
```
Comment (amousset) [inline webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala]: :thinking:

**Comment by VinceMacBuche** [comment]
There is to commits, can you rebase it ? or edit commit messageof second on to be a fixup!

File: webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala (hunk 1 above)
Comment (VinceMacBuche) [reply webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala]: :boom:

Comment (VinceMacBuche) [reply webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/batch/AsyncDeploymentActor.scala]: I think you did not want to commit that file and was automatically commited by rudder-dev

**Comment by ncharles** [comment]
rudder-dev fails, and pushed extra files the second time. Now github doesn't refresh the code :)

**Comment by VinceMacBuche** [comment]
It's not changing expected result of policy generation tests ? I thought it would change in rudder.json but maybe not ...

IF tests are ok it's good to me!

**Comment by ncharles** [comment]
Tests are all great for me

**Pull request merged**